sormas-ui/src/main/java/de/symeda/sormas/ui/labmessage/LabMessageController.java

```diff
@@ -310,7 +312,10 @@ private void pickOrCreatePerson(LabMessageDto labMessage) {
 
 					pickOrCreateEntry(labMessage, similarCases, similarContacts, similarEventParticipants, selectedPersonDto);
 				}
-			}, false);
+			},
+				false,
+				personSearchActive,
+				personSearchActive ? I18nProperties.getString(Strings.infoSelectOrCreatePersonForLabMessageWithoutMatches) : null);
```

Comment:
The call of the method ControllerProvider.getPersonController().selectOrCreatePerson(...) is 38 lines long now.
This is hardly readable. Maybe define the resultConsumer before the method call, and split into several parts with meaningful names.

Reply:
I don't see any related changes in the last commit.